harness/determined/exec/harness.py

```diff
@@ -149,6 +132,6 @@ def main(train_entrypoint: Optional[str]) -> int:
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
-    parser.add_argument("--train-entrypoint")
+    parser.add_argument("train-entrypoint")
```

Comment:
IIRC, i think this "--" is needed by argparse if you have a "-" in the argument name.